### PR TITLE
Add new Admin Structures (Templates for Events)

### PR DESCRIPTION
**Pull request by Venuska1117**
## What Does This PR Do
Add new category to _Map Template - Place_ list, this list for now contain 2 structures with were made for testing _(but they are made for events)_ they have |Event| in beginning of name.

## Why It's Good For The Game
Add quick "Templates" for Admin Events, some admins like to make events, like thunderdome fight, or some title screen sanbox, so i tought, why not make it more easy for admins to make them more often with less work by using templates, and allow people to add more in future. _(QoL for Admins)_

## Images of changes

There is example of "splashscreen sandbox" template.
sandbox time baby

Templates can be improved, they made for testing and demonstration _(but can be used in future)_

## Testing
Run code, tested it, and it WORK _(amazing)_

## Changelog
:cl:
add: Add Admins "Event Structures" Templates in "Map Template" verb.
/:cl:

### CODE REVIEW

**Comment by Sadhorizon** [comment]
a few things:
1. I'd add some paper bins, to pair with the tape. It will allow making cool decorations. As well as a couple of pens maybe, to further add to that goal. Paper decorations can work in a lot of places.
2. Unsure what the nanopaste is for in there. I guess IPCs cannot use the medibot, but also they get a lot of tools already.
3. Plasmamen/vox gear is rather far away from the cloner. Might want to move it closer so they don't immediately die/set everything on fire when going there.

File: code/datums/event_structures.dm [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+/datum/map_template/event/splashscreen_empty
+	name = "Splashscreen Empty"
+	event_id = "|Event| Splashscreen Empty"
+	description = "Empty starting area, good for custom admin game-join screen or admin foolery."
+	mappath = "_maps/map_files/EventStructures/splashscreen_empty.dmm"
+
+/datum/map_template/event/splashscreen_sandbox
+	name = "Splashscreen Sandbox"
+	event_id = "|Event| Splashscreen Sandbox"
+	description = "Pre-made starting area with contain respawner, make sure to use ghost beacon, it contain all basic sandbox tools necessery for your ants to create colony."
```
Comment (Henri215) [inline code/datums/event_structures.dm:14]:
```suggestion
	description = "Pre-made starting area that contains a respawner, make sure to use the ghost beacon. It has all the basic sandbox tools necessary for your ants to create colony."
```

Comment (Venuska1117) [reply code/datums/event_structures.dm:14]: splendid

**Comment by Venuska1117** [comment]
> a couple things:
> 
> 1. I'd add some paper bins, to pair with the tape. It will allow making cool decorations. As well as a couple of pens, to further add to that goal.
> 2. Plasmamen/vox gear is rather far away from the cloner. Might want to move it closer so they don't immediately die/set everything on fire when going there.

Done both, now breathing Gear is closer to "respawning" pod

**Review by S34NW — APPROVED** [review]

**Pull request merged**